phys2bids/tests/test_txt.py

```diff
@@ -26,7 +26,6 @@ def loaded_lab_file(multifreq_lab_file):
     header_lab, channels_lab = txt.read_header_and_channels(multifreq_lab_file, chtrig)
 
     # just a few quick checks to make sure the data loaded correctly
-    assert len(channels_lab[152109 - 9]) == 6  # check the comment has been eliminated
```

Comment:
Well, we might have a coverage problem here. This assert was checking that a comment in the file was eliminated, therefore i suggest you add a comment to the test_file in the channels part and then when loading the file make sure that is delete

Reply:
I noticed it and removed the line since the new file doesn't have a comment. 

If that's a change that can be done quickly, do you think you could do that? I have no idea how you did it last time. If it's not a quick fix, I'll do it.

Reply:
what's the file you are using?

Reply:
Actually if you check the coverage report is the only change. Could you tell me which file are you using?